### DISCUSSION TITLE
Which port does AndroRAT use?

### DISCUSSION THREAD

**Discussion by ghost** [karma9874/AndroRAT#375] [Q&A] [answered]
I am trying to ask which port does AndroRAT use, or it uses the port you've set?

**Comment by Youngthugshsfvsihwiwiwhsbs** [comment]
Hello

**Reply by ghost** [reply]
can you please tell me what port does androrat use? or it use port i setted in building?

**Reply by takakael** [reply, marked as answer]
String IP = "192.168.0.105";
5static String port = "8888";
ostatic boolean icon = true;

But i guess you configure that yourself 


Android_Code/app/src/main/java/com/example/reverseshell2/config.java

**Reply by ghost** [reply]
Okay, thanks.
